src/serial.rs

```diff
@@ -1,3 +1,41 @@
+//! # Serial Communication
+//! This module contains the functions to utilize the USART (Universal
+//! synchronous asynchronous receiver transmitter (USART)
```

Comment:
```suggestion
//! synchronous asynchronous receiver transmitter)
```
Small duplication here.